act-rules-format.bs

```diff
@@ -15,86 +15,112 @@ Markup Shorthands: markdown yes
 
-* Descriptive title
-* [Unique identifier](#unique-identifier)
-* [Rule Description](#structure-description)
-* [Accessibility Requirements](#structure-accessibility-requirements)
-* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
-* [Accessibility Support](#acc-support) (optional)
-* [Aspects Under Test](#input-aspects)
-* [Test Definition](#test-def)
+- [Composed rules](#composed-rule-outline) describe how results from atomic rules should be used to decide if a an accessibility requirement was failed. If there are multiple failures necessary for an accessibility requirement to fail, each "type" of failure should be written as separate atomic rules and be aggregated by a composed rule.
 
-Unique Identifier {#unique-identifier}
-----------------------------------------
+The separation between atomic rules and composed rules creates a division of responsibility. Atomic rules test if web content correctly implemented a particular solution. Composed rules test if a combination of `pass` outcomes from atomic rules is sufficient for an accessibility requirement not to `fail`. Not all atomic rules have to be part of a composed rule. Only if failing that atomic rule does not directly fail an [Accessibility Requirements](#structure-accessibility-requirements). An atomic rule MAY be part of multiple composed rules.
 
-An ACT Rule MUST have a unique identifier that can be any unique text value, such as plain text, URL or a database identifier.
+A composed rule defines how the outcomes from its atomic rules are aggregated into a single outcome for each applicable test target. Note that atomic rules in a composed rule MAY have different applicability. Because of this, not every element applicable within the composed rule is tested by every atomic rule. Atomic rules MAY also be disabled during a test run due to accessibility support concerns. See [Accessibility support](#acc-support) for details.
 
+<div class="example">
+  <p>Composed rule: Header cells in HTML tables (WCAG 2 Success Criterion 1.3.1)</p>
+  <p>This rule would use results from the following atomic rules:</p>
+  <ul>
+    <li>Header indicated through implicit scope</li>
+    <li>Header indicated through the `scope` attribute</li>
+    <li>Header indicated by using the `headers` attribute</li>
+    <li>Header indicated by using ARIA labels</li>
+  </ul>
+  <p>If any one of these rules passes, the table cell can pass the composed rule.</p>
+</div>
 
-Rule Description {#structure-description}
+
+ACT Rule Structure {#act-rule-structure}
+===============================
+
+Atomic Rule Outline {#atomic-rule-outline}
 ----------------------------------------
 
-An ACT Rule MUST have a description that is in plain language and provides a brief explanation of what the rule does.
+An Atomic ACT Rule MUST consist of the following items:
```

Comment:
I suggest merging the Atomic and Composed rule outlines into one, making it clearer how the two rule types differ, e.g.:

"An ACT Rule MUST consist of the following items:
 	 
* Descriptive Title
* [Rule Identifier](#rule-identifier)
* [Rule Description](#structure-description)
* [Accessibility Requirements](#structure-accessibility-requirements)
* [Aspects Under Test](#input-aspects) (for atomic rules) OR [Atomic Rules List](#atomic-rules-list) (for composed rules)
* [Test Definition](#test-def) (for atomic rules) OR [Aggregation Definition](#aggregation-definition) (for composed rules)
+* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
* [Accessibility Support](#acc-support) (optional)"

Reply:
Atomic rules doesn't necessarily have any accessibility requirements that fail if the rule fails, so I suggest adding "if any" to the Accessibility Requirements item:

"An ACT Rule MUST consist of the following items:
 	 
* Descriptive Title
* [Rule Identifier](#rule-identifier)
* [Rule Description](#structure-description)
* [Accessibility Requirements](#structure-accessibility-requirements), if any
* [Aspects Under Test](#input-aspects) (for atomic rules) OR [Atomic Rules List](#atomic-rules-list) (for composed rules)
* [Test Definition](#test-def) (for atomic rules) OR [Aggregation Definition](#aggregation-definition) (for composed rules)
* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
* [Accessibility Support](#acc-support) (optional)"

Is this already reflected in section linked to from here?